pkg/dispatcher/dispatcher.go

```diff
@@ -100,16 +103,45 @@ func (p *Dispatcher) dispatch(event events.SchedulingEvent) error {
 	if !p.isRunning() {
 		return fmt.Errorf("dispatcher is not running")
 	}
-	for p.isRunning() {
-		select {
-		case p.eventChan <- event:
-			return nil
-		case <-time.After(time.Second * 3):
-			log.Logger.Error("event channel is full, keep waiting",
-				zap.Int("channelLength", len(p.eventChan)))
-		}
+	select {
+	case p.eventChan <- event:
+		log.Logger.Info("dispatched", zap.Int("capacity", cap(p.eventChan)),
```

Comment:
I think we can remove this logging, otherwise even debug it would be overwhelming

Reply:
Sure, this is added for local testing, forgot to remove.